requirements.txt

```diff
@@ -15,3 +15,4 @@ PyYAML==5.3
 six==1.14.0
 tornado==6.0.3
 mkdocs-macros-plugin==0.4.18
+mkdocs-awesome-pages-plugin=2.5.0
```

Comment:
```$ make .venv
Creating a virtualenv in .venv... 
Installing packages in .venv... 
ERROR: Invalid requirement: 'mkdocs-awesome-pages-plugin=2.5.0' (from line 18 of requirements.txt)
Hint: = is not a valid operator. Did you mean == ?
make: *** [Makefile:107: .venv] Error 1
```

Reply:
Time to go for an eye check up.

Fixed.